src/anshuKJun20/ReverseNumber.java

```diff
@@ -0,0 +1,24 @@
+package anshuKJun20;
+
+public class ReverseNumber {
+	int add = 0;
+
+	void dispalyReverseNumber(int number) {
+		int number1 = number;
+		while (number > 0) {
+			int remainder = number % 10;
+			int quotient = number / 10;
+			number = quotient;
```

Comment:
Instead of writing number = quotient ;you can directly store value in number variable
int number=number/10;